runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesOverlay.java

```diff
@@ -95,9 +95,9 @@ public Dimension render(Graphics2D graphics)
 		switch (action)
 		{
 			case ITEM_USE_ON_WIDGET:
-				if (!menuEntry.getTarget().contains("High Level Alchemy") || !config.showAlchProfit())
+				if (!config.showWhileAlching() || !menuEntry.getOption().equals("Cast") || !menuEntry.getTarget().contains("High Level Alchemy"))
 				{
-					break;
+					return null;
```

Comment:
There is no reason to change this to return null when you can just use break like it is atm on master